packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -376,6 +377,30 @@ export const HelpCenterContactForm = () => {
 		}
 	};
 
+	const getSupportedLanguages = ( supportType: string, locale: string ) => {
+		const isLiveChatLanguageSupported = (
+			config( 'livechat_support_locales' ) as Array< string >
+		 ).includes( locale );
+
+		const isLanguageSupported = ( config( 'upwork_support_locales' ) as Array< string > ).includes(
+			locale
+		);
+
+		switch ( supportType ) {
+			case 'CHAT':
+				return ! isLiveChatLanguageSupported;
+			case 'EMAIL':
+				return ! isLanguageSupported && ! [ 'en', 'en-gb' ].includes( locale );
+
+			default:
+				return false;
+		}
+	};
+
+	const shouldShowHelpLanguagePrompt = () => {
+		return getSupportedLanguages( mode, locale );
+	};
```

Comment:
I think you should do 
```suggestion
	const shouldShowHelpLanguagePrompt = getSupportedLanguages( mode, locale ); 
```